src/Services/SwaggerService.php

```diff
@@ -804,9 +809,19 @@ public function saveProductionData()
 
     public function getDocFileContent()
     {
-        $documentation = $this->driver->getDocumentation();
+        try {
+            $documentation = $this->driver->getDocumentation();
+
+            $this->openAPIValidator->validate($documentation);
+        } catch (\Exception $exception) {
+            $data = $this->generateBaseDataObject();
 
-        $this->openAPIValidator->validate($documentation);
+            $this->config['info'] = Arr::set($this->config, 'info.description', Arr::get($this->config, 'defaults.error'));
```

Comment:
let's pass the description view as arg, it would be better to redefine the config field